packages/core/src/components/tabs/inline-tabs/inline-tabs.tsx

```diff
@@ -32,6 +32,9 @@ export class TdsInlineTabs {
    * If this is set, all Tab changes need to be handled by the user. */
   @Prop({ reflect: true }) selectedIndex: number;
 
+  /** Custom left padding value for the wrapper element. */
+  @Prop({ reflect: true }) leftPadding: number = 0;
+
```

Comment:
```suggestion
  /** Custom left padding value for the wrapper element. */
  @Prop({ reflect: true }) leftPadding: number = 32;

```

To avoid braking change: 
We need to keep the visual component unchanged for current users. 
User that goes from version 1.13.0 to a newer one, should not see any visual difference between therefor this small change.
Please update that and the screenshots for the unit test as they might fail.